internal/config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+import (
+	"os"
+	"path"
+
+	"github.com/layer5io/meshery-adapter-library/config"
+	configprovider "github.com/layer5io/meshery-adapter-library/config/provider"
+	"github.com/layer5io/meshkit/utils"
+)
+
+const (
+	LinkerdOperation = "linkerd"
+	Development      = "development"
+	Production       = "production"
```

Comment:
Never quite sure whether it is a good thing to have an application be aware of the environment it is running in. I would avoid it if at all possible.
Why was this approach chosen rather than another one, and which other ones were considered?
How is this going to be extended with other environments, like test, staging, etc?

Reply:
I do not have an answer for this @mgfeller. We can choose to get rid of it, or have it. Lets remove this @utkarsh-pro and call it just call it `Config` object.

Reply:
Yeah, that makes sense. Will make the changes.

Reply:
We could use an environment variable to point to a possibly remote config file overriding the default config for different environments etc, and have Viper fetch it (if supported), or download using GetFileResource :smile: and have Viper load it.

Reply:
"Environments" are a concept yet to come into being in Meshery, but are on the roadmap. Until they are designed, let's avoid building them in anywhere, if we can.